sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -104,7 +104,7 @@ def __init__(self, client: "EventHubConsumerClient", source: str, **kwargs: Any)
         link_properties: Dict[bytes, int] = {}
         self._error = None
         self._timeout = 0
-        self._idle_timeout = (idle_timeout * self._amqp_transport.TIMEOUT_FACTOR) if idle_timeout else None
+        self._idle_timeout: Optional[int] = (idle_timeout * self._amqp_transport.TIMEOUT_FACTOR) if idle_timeout else None
```

Comment:
have we used float for timeout or this one always int?

Reply:
just had to scroll a bit, its always been int. Have we been uniform about this?

Reply:
nope its float someplaces and int elsewhere